lib/esConnector.js

```diff
@@ -939,7 +939,8 @@ ESConnector.prototype.destroy = function destroy(modelName, id, done) {
  * NOTES:
  * > The _source field need to be enabled for this feature to work.
  */
-ESConnector.prototype.updateAttributes = function updateAttrs(modelName, id, data, callback) {
+
+function updateAttrs(modelName, id, data, options, callback) {
```

Comment:
@juanpujol - if you agree with what you see below then can you also commit the following for backward compatibility:

```
// pseudo-code:
// if options is a function (not an object) then set `callback=options` and `options=null`
```

let me know